back/node_watcher/kubernetes/staging/nomidotwatcher-job.yaml

```diff
@@ -8,23 +8,20 @@ metadata:
     name: nomidotwatcher
     app: nomidotwatcher
 spec:
-  selector:
-    matchLabels:
-      app: nomidotwatcher
-  replicas: 1
-  strategy:
-    type: RollingUpdate
   template:
     metadata:
       labels:
         stage: staging
         name: nomidotwatcher
         app: nomidotwatcher
     spec:
+      restartPolicy: Never
       containers:
         - name: nomidotwatcher
-          image: eu.gcr.io/test-installations-222013/nomidot_watcher@sha256:73f32773eb4e397e9f4cc7028945750f8f1116c7e751179120300a18413797ee
+          image: eu.gcr.io/test-installations-222013/nomidot_watcher@sha256:23bda9bd84369d74bc4f9a299649f388e4676bf37700c82a2df2cb577808d00a
           env:
             - name: PRISMA_ENDPOINT
               value: http://10.0.9.43:4466
+            - name: START_FROM
+              value: '38000'
```

Comment:
this way I don't have to create and push a whole new image everytime i want to recover from a failure from a particular branch. just need to do `kubectl replace -f ...`